### ISSUE TITLE
querying using WhereOperator.IN passing null list returns NullPointerException

### ISSUE THREAD

**Issue by danilodeLuca** [feroult/yawp#28]
```
java.lang.NullPointerException
	at io.yawp.repository.query.condition.SimpleCondition.assertIsList(SimpleCondition.java:113)
	at io.yawp.repository.query.condition.SimpleCondition.<init>(SimpleCondition.java:30)
	at io.yawp.repository.query.condition.Condition.c(Condition.java:11)
	at io.yawp.repository.query.condition.Condition.c(Condition.java:7)

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at io.yawp.repository.hooks.RepositoryHooks.invokeHookMethod(RepositoryHooks.java:51)
	at io.yawp.repository.hooks.RepositoryHooks.invokeHooks(RepositoryHooks.java:35)
	at io.yawp.repository.hooks.RepositoryHooks.beforeQuery(RepositoryHooks.java:22)
	at io.yawp.repository.Repository.queryWithHooks(Repository.java:144)
	at io.yawp.repository.Feature.yawpWithHooks(Feature.java:18)

```

**Comment by luanpotter** [comment]
What would be expected? No records, all records...

**Comment by danilodeLuca** [comment]
Treat as an empty list, no results would be expected.
Think about not executing query or this condition(if its "or" query), to not waste cost querying on GAE

**Comment by luanpotter** [comment]
Fixed in 0337c10. It already worked the way described with smpty lists, I just added to the condition :)
